spec/xclarity_client_unmanage_request_spec.rb

```diff
@@ -0,0 +1,43 @@
+require 'spec_helper'
+
+describe XClarityClient do
+
+  before :all do
+    WebMock.allow_net_connect! #-- Uncomment this line if you're testing with a external mock.
+
+    conf = XClarityClient::Configuration.new(
+    :username => ENV['LXCA_USERNAME'],
+    :password => ENV['LXCA_PASSWORD'],
+    :host     => ENV['LXCA_HOST'],
+    :port     => ENV['LXCA_PORT'],
+    :auth_type => ENV['LXCA_AUTH_TYPE'],
+    :verify_ssl => ENV['LXCA_VERIFY_SSL']
+    )
+
+    @client = XClarityClient::Client.new(conf)
+
+    @host = ENV['LXCA_HOST']
+  end
+
+  it 'has a version number' do
+    expect(XClarityClient::VERSION).not_to be nil
+  end
+
+  describe 'GET /unmanageRequest/jobs/<job_id>' do
+    it 'should return a non empty array' do
+      response = @client.fetch_unmanage_request('4685')
+      expect(response).not_to be_empty
+    end
```

Comment:
Could you ensure that part of data is the same of the data defined in the blue print API? I mean, check if the data are coming.

Reply:
Added an expect statement to check if the data defined in the blueprint is what is returned by the client API.